design-documents/graph-ql/coverage/requisitionList.graphqls

```diff
@@ -0,0 +1,261 @@
+## Gift Card product type support will be added after the Product Implementation is developed
+type Query {
+    "The requisitionList query returns the contents of customer's  Requisition List"
+    requisitionList(
```

Comment:
Should any requisition-related queries be accessible by the guest users? If not, they should be moved under customer.

Reply:
Requisition lists are only available for customer accounts, not guests